pkg/asset/installconfig/powervs/metadata.go

```diff
@@ -278,7 +273,7 @@ func (m *Metadata) EnsureVPCIsPermittedNetwork(ctx context.Context, vpcName stri
 
 // GetSubnetID gets the ID of a VPC subnet by name and region.
 func (m *Metadata) GetSubnetID(ctx context.Context, subnetName string, vpcRegion string) (string, error) {
-	subnet, err := m.client.GetSubnetByName(ctx, subnetName, vpcRegion)
+	subnet, err := m.sessionClient.GetSubnetByName(ctx, subnetName, vpcRegion)
```

Comment:
Same thing here. But in this case you should use `client()` since `GetSubnetID` could be called with a unitialized `sessionClient` by users of `metadata`.

Reply:
Understood, but perhaps by luck, that's not happening now. When that happens for a new scenario, you/developer get a segfault, find out `.sessionClient=nil`, then find `.client()` func, and start calling it (along with other necessary adjustments) now that you have an actual crash to fix and have to do it?

Reply:
PTAL at the latest force-push.